src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "ms-blazorwasm-companion",
+  "displayName": "Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension",
+  "description": "Companion extension for debugging Blazor WebAssembly",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.52.0"
+  },
+  "categories": [
```

Comment:
Definitely will be curious if we can suppress this from the marketplace

Reply:
Chatted with someone on the marketplace team about this. They don't currently support unlisting extensions.

However, they recommended that we make the companion extension dependent on the C# extension so that the user is never at risk of installing the companion extension without its parent so I've done that here.

Reply:
ahhh, that's interesting, so basically both O# and us depend on each other.